<ui version="4.0" >
 <class>TurnDialog</class>
 <widget class="QDialog" name="TurnDialog" >
  <property name="windowModality" >
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>232</width>
    <height>164</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string> Panini - Orient Image</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>121</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="RollBox" >
   <property name="geometry" >
    <rect>
     <x>130</x>
     <y>50</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="accelerated" >
    <bool>true</bool>
   </property>
   <property name="minimum" >
    <double>-45.000000000000000</double>
   </property>
   <property name="maximum" >
    <double>45.000000000000000</double>
   </property>
   <property name="singleStep" >
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="PitchBox" >
   <property name="geometry" >
    <rect>
     <x>130</x>
     <y>80</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="accelerated" >
    <bool>true</bool>
   </property>
   <property name="minimum" >
    <double>-90.000000000000000</double>
   </property>
   <property name="maximum" >
    <double>90.000000000000000</double>
   </property>
   <property name="singleStep" >
    <double>0.200000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>54</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Roll degrees</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>84</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Pitch degrees</string>
   </property>
  </widget>
  <widget class="QComboBox" name="TurnList" >
   <property name="geometry" >
    <rect>
     <x>110</x>
     <y>20</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3" >
   <property name="geometry" >
    <rect>
     <x>40</x>
     <y>20</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text" >
    <string>Orientation</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TurnDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TurnDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
